packages/bpk-component-input/src/BpkInput.tsx

```diff
@@ -149,16 +137,16 @@ class BpkInput extends Component<Props, State> {
             label={clearButtonLabel || ''}
             onMouseDown={onMouseDown}
             onClick={(e) => {
-              if (ref) {
-                ref.focus();
-              }
               if (onClear) {
+                if (ref) {
```

Comment:
I meant that the ref focus was moved inside the if. So previously it would still be focused even if there was no `onClear`, but yeah you're right, this should be good!